src/lumigo_tracer/utils.py

```diff
@@ -307,3 +308,7 @@ def omit_keys(value: Any):
             for k, v in value.items()
         }
     return value
+
+
+def warn_client(msg: str) -> None:
+    print(f"{WARN_CLIENT_PREFIX}: {msg}")
```

Comment:
he should have the ability to turn off this kind of messages - do it with env_key (copy from the node tracer, it is already defined there)